content/blog/2019-11-roxygen2-7-0-0/index.Rmarkdown

```diff
@@ -0,0 +1,281 @@
+  matches the intended usage of the `\code{}` and `\verb{}` macros and should 
+  make it easier to include arbitrary "code" snippets in documentation without 
+  causing Rd failures.
+
+* `@family` automatically adds `()` when linking to functions,
+  and prints each link on its own line (to improve diffs).
+  
+We've also removed a few features to simplify the code and/or clearly advertise that certain features are no longer supported:
+
+* `@S3method` has been removed. It was deprecated in roxygen2 4.0.0
+  released 2014-05-02, over 5 years ago.
+
+* Using the `wrap` option will now trigger a warning, as it hasn't worked
+  for quite some time. Suppress the wraning by remove the option from your
+  `DESCRIPTION`.
+
+* `@docType package` will no longer add `-name` to the file name. If you relied 
+  on this, either switch to the 
+  [new workflow](https://roxygen2.r-lib.org/articles/rd.html#packages), or
+  use `@name` to manaully override the default file name.
+
+## Markdown improvements
+
+There are two major improvements to roxygen2's markdown support: you can now use markdown headers and tables.
+
+Markdown headings can be used in top-level tags like `@description`, `@details`, and `@returns`. Level 1 headings create a new top-level `\section{}` and Level 2 headings (and below) create nested `\subsection{}`s:
+
+```R
+#' @details
+#' First sentence
+#'
+#' ## Subheading
+#' Second sentence
+#'  
+#' # New section
+#' Third sentence
+```
+
+is translated to
+
+```
+\details{
+  First sentence
+  
+  \subsection{Subheading}{
+    Second sentence
+  }
+}
+\section{New section}{
+  Third sentence
+}
+```
+
+Markdown tables used the [GFM table syntax](https://github.github.com/gfm/#tables-extension-) and are converted to `\tabular{}` macros. For example,
+
+```R
+#' | foo | bar |
+#' | :-- | --: |
+#' | baz | bim |
+```
+
+is translated to
+
+```
+\tabular{lr}{
+   foo \tab bar \cr
+   baz \tab bim \cr
+}
+```
+
+Lastly, using unsupported markdown features (like blockquotes, inline HTML, and horizontal rules) will now produce an informative message.
+
+## `@includeRmd`
+
+`@includeRmd` provides a new tool that allows you to share text and code amongst `.Rmd`  and `.Rd`. For example, if you have `vignettes/common.Rmd`, you can include it:
+
+*   In documentation, with:
+
+    ````
+    #' @includeRmd vignettes/common.Rmd
+    ````
+  
+*   In other vignettes with:
+
+    ````
+    ```{r child = "common.Rmd"}`r ''`
+    ```
+    ````
+    
+*   In `README.Rmd`, with:
+
+    ````
+    ```{r child = "vignettes/common.Rmd"}`r ''`
+    ```
+    ````
+
+Learn more in 
+[`vignette("rd")`](https://roxygen2.r-lib.org/articles/rd.html#including-external--rmd-md-files)
+
+## R6 documentation
+
+You can now document R6 classes! The basic usage is straighforward and works similarly to documenting function. The main difference is that methods require explicit `@description` and `@detail` tags.
```

Comment:
"documenting functions"

Reply:
You can make suggestions directly (like Mara did) for typos